com.github.Qv2ray.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=com.canonical.Unity",
+        "--talk-name=org.freedesktop.portal.Fcitx",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "grpc",
+            "no-autogen": true,
+            "make-install-args": [
+                "prefix=/app"
```

Comment:
I'm fairly sure it's the default. You shouldn't need it here or anywhere below.

Reply:
Never mind, just noticed your comment.

Reply:
Hey! I am not sure what you mean. Do I need to change something?

Reply:
You can remove the make-install-args here as the prefix is already set to /app by default.

Reply:
![image](https://user-images.githubusercontent.com/39402395/72330720-f72a8a80-36f1-11ea-8acb-d44a9d2672a3.png)
Not working for grpc.